web/views.py

```
@@ -1301,29 +1301,57 @@
+    # Prepare context with students and their attendance status
+    students_data = []
+    for enrollment in enrollments:
+        student = enrollment.student
+        attendance = attendances.get(student.id)
+
+        students_data.append(
+            {
+                "student": student,
+                "attendance": attendance,
+                "is_pending": attendance.status == "pending" if attendance else False,
+            }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)